esprima.js

```diff
@@ -187,6 +188,7 @@
         IllegalContinue: 'Illegal continue statement',
         IllegalBreak: 'Illegal break statement',
         IllegalReturn: 'Illegal return statement',
+        IllegalYield: 'Illegal yield expression',
```

Comment:
It's cosmetics and not critical. Once V8 has the error message that we want, we can adopt it. In all cases, the consumer will need to adjust accordingly. It's just easier to transition to a specialized V8-compatible message from a generic "Unexpected token..."  than from an arbitrary one.